OdeToFood/Models/OdeToFoodDB.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using OdeToFood;
+
+namespace OdeToFood.Models
+{
+    public class OdeToFoodDB
+    {
+        public IEnumerable<Restaurant> Restaurants {
+            get
```

Comment:
Zoveel logica in een get stoppen is niet verstandig want dan wordt het elke keer opnieuw uitgevoerd wanneer iemand de property Restaurants aanroept.. Verstandiger is om hier een echte method van te maken.

Reply:
Properties mogen eigenlijk alleen 'snelle' acties zijn.

Reply:
Johan Piet had dezelfde suggestie inderdaad. Ik heb het aangepast in de code.